src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -152,19 +150,23 @@ public void test() throws IOException { //NOPMD - assert is done in subs.
         playerDies();
         clickPopup(DEATH_TOMAIN);
 
-//        mainMenu(MAIN_QUIT);
+        //        mainMenu(MAIN_QUIT);
         Platform.runLater(stage::close);
         outputEventLog();
         new File("sav/Fred.ses").delete();
     }
 
-    private void outputEventLog() throws IOException {
-        ShaftEscape.LOGGER.writeToFile();
-        final String log = new String(Files.readAllBytes(
-                Paths.get(Logger.OUTFILE)), StandardCharsets.UTF_8);
-        System.out.println("\n== EVENT_LOG ==");     //NOPMD - Intended use of
-        System.out.println(log);                     //NOPMD - System.out.print
-        System.out.println("== END_EVENT_LOG ==\n"); //NOPMD - for Travis log
+    private void outputEventLog() {
+        try {
+            ShaftEscape.LOGGER.writeToFile();
+            final String log = new String(Files.readAllBytes(
+                    Paths.get(Logger.OUTFILE)), StandardCharsets.UTF_8);
+            System.out.println("\n== EVENT_LOG ==");     //NOPMD - Intended use of
+            System.out.println(log);                     //NOPMD - System.out.print
+            System.out.println("== END_EVENT_LOG ==\n"); //NOPMD - for Travis log
+        } catch (IOException e) {
+            fail();
```

Comment:
No I do not agree. It is not acceptable if this exception makes it to the test. It should be caught earlier in a method. Any exception that can be seen by a test (which acts as the user in this case) is wrong and should result in a failure.

The whole point of catching exceptions is that the user is not able to see that something is going wrong. If he does see something, that means some components are failing

< review, logic